packages/@sparrow-common/src/components/header/Header.svelte

```diff
@@ -84,6 +97,28 @@
     },
   ];
 
+  let multipleAgentData = [
+    {
+      name: "Cloud Agent",
+      id: "Cloud Agent",
+      displayName: "Cloud Agent",
+      description:
```

Comment:
This description is not completely appropriate. Let's discuss on this.

Reply:
Can we add the below line. 

Ideal for bypassing CORS limitations of browsers.